src/server/player_sao.cpp

```diff
@@ -618,7 +618,7 @@ bool PlayerSAO::checkMovementCheat()
 	float player_max_jump = 0; // vertical upwards movement
 
 	float speed_walk = m_player->movement_speed_walk * m_player->physics_override.speed;
-	float speed_fast = m_player->movement_speed_fast;
+	float speed_fast = m_player->movement_speed_fast * m_player->physics_override.speed;
 	float speed_crouch = m_player->movement_speed_crouch * m_player->physics_override.speed_crouch;
```

Comment:
https://github.com/minetest/minetest/blob/6106e4e72b5d337819b16d7aace52610d948131c/src/client/localplayer.cpp#L694-L697

All speeds are affected, including crouch. It might make sense to first figure out the maximal speed and then apply the speed modifier on top of that. Like it was done before the PR:
https://github.com/minetest/minetest/commit/8ebaf753d3c6b50#diff-60afc4b1ee298e01efae55db3b202d0bdbffd3919d7f0bc3212a4d300522eacaL616

Reply:
I've now applied the speed override where applicable: walk, crouch, fast, and climb

(I've decided against multiplying afterwards for consistency with max jump, in which case the speed override is not applied for the maximum of everything, but just for climb.)